testsuite/dockerfiles/container-registry/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM registry:2
```

Comment:
Just curious, what is exactly behind the number 2? How does Docker know what image is behind this number?

Reply:
It is registry version 2